README.md

```diff
@@ -0,0 +1,47 @@
+# BitlBee
+
+![](http://bitlbee.org/style/logo.png)
+
+[![Build Status](https://travis-ci.org/bitlbee/bitlbee.svg)](https://travis-ci.org/bitlbee/bitlbee)
+[![Coverity Scan Build Status](https://scan.coverity.com/projects/4028/badge.svg)](https://scan.coverity.com/projects/4028)
+
+An IRC to other chat networks gateway
+
+Main website: http://www.bitlbee.org/
+
+Bug tracker: http://bugs.bitlbee.org/
+
+Wiki: http://wiki.bitlbee.org/
+
+License: GPLv2
+
+## Development
+
+Use github pull requests to submit patches.
+
+Mappings of bzr revisions to git commits: [html][], [raw][] (created using [git-bzr-rev-map][])
```

Comment:
Actually, I just updated it today to cover up to 1072. It's mostly there for historical purposes, such as old comments referencing bzr revisions